addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15919,7 +15929,7 @@ msgstr ""
 #. label of a setting for the stereoscopic 3D mode of the GUI that is/should be applied
 #: system/settings/settings.xml
 msgctxt "#36500"
-msgid "Stereoscopic 3D mode (current)"
+msgid "Current stereoscopic 3D mode"
```

Comment:
@da-anda OK have done back to original, But in that case why not.
```
"Switch 3D mode and/or show current."
```
I think we gone ape using stereoscopic 3D, monoscopic 2D all over the place, 2D/3D should be enough no?